projects/markdown/README.md

```diff
@@ -5,6 +5,11 @@
 This is fork of popular library `marked` from [this commit](https://github.com/markedjs/marked/tree/39fbc8aed)
 (Merge pull request #961 from chjj/release-0.3.7, Dec 1, 2017).
 
+## lang
+- [chinese](./lang/zh/README.md)
```

Comment:
You need rename file `lang/zh/README_zh.md` (in this Pull Request) to `lang/zh/README.md` as you write here in the docs.